.github/workflows/ci.yml

```diff
@@ -7,6 +8,15 @@ on:
     branches:
       - master
     tags: '*'
+  merge_group:
+    types: [checks_requested]
```

Comment:
I think you can omit this line.
```suggestion
```

Reply:
Sure, we can. But my interpretation of the [Github docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#merge_group) was that it would be safer to state the `types` explicitly:

> More than one activity type triggers this event. Although only the checks_requested activity type is supported, specifying the activity type will keep your workflow specific if more activity types are added in the future.